pkgs/applications/audio/friture/default.nix

```
@@ -1,33 +1,35 @@
-{ lib, fetchFromGitHub, python3Packages, wrapQtAppsHook }:
+{ lib, fetchFromGitHub, fetchpatch, python3Packages, wrapQtAppsHook }:
 
 let
   py = python3Packages;
 in py.buildPythonApplication rec {
   pname = "friture";
-  version = "0.37";
+  version = "unstable";
```

current convention is 
```suggestion
  version = "unstable-YYYY-MM-DD";
```
Where the date is that of the commit